src/manage_sql_nvts.c

```diff
@@ -1455,6 +1461,35 @@ update_nvts_from_vts (entity_t *get_vts_response,
           count_new_vts, count_modified_vts);
 
   sql_commit ();
+
+  if (osp_vt_hash && strcmp (osp_vt_hash, ""))
+    {
+      char *db_vts_hash;
+
+      db_vts_hash
+        = sql_string ("WITH str AS"
+                      "  (SELECT oid||modification_time as vt_string"
+                      "   FROM nvts ORDER BY oid)"
+                      " SELECT encode(digest(string_agg(str.vt_string,''),"
+                      "                      'sha256'),"
+                      "               'hex')"
+                      " FROM str;");
+
+      if (strcmp (osp_vt_hash, db_vts_hash ? db_vts_hash : ""))
+        {
+          g_warning ("%s: SHA-256 hash of the VTs in the database does not"
+                     " match the one from the scanner.",
+                     __func__);
+          g_debug ("%s: Hash from OSP: %s, from database: %s",
```

Comment:
Any reason for not putting this in the warning?  Could be useful to support.

Reply:
I thought this may only be useful for debugging but I don't mind combining the two, so
I've done it in aaa7768402a4371ccf407ea221647144105817d4